src/target/riscv/program.h

```diff
@@ -5,17 +5,25 @@
 
 #include "riscv.h"
 
-#define RISCV_MAX_DEBUG_BUFFER_SIZE 32
+#define RISCV_MAX_PROGBUF_SIZE 32
 #define RISCV_REGISTER_COUNT 32
 #define RISCV_DSCRATCH_COUNT 2
 
+typedef enum {
+	RISCV_DBGBUF_EXEC_RESULT_NOT_EXECUTED,
+	RISCV_DBGBUF_EXEC_RESULT_UNKNOWN,
+	RISCV_DBGBUF_EXEC_RESULT_EXCEPTION,
+	RISCV_DBGBUF_EXEC_RESULT_UNKNOWN_ERROR,
+	RISCV_DBGBUF_EXEC_RESULT_SUCCESS
+} riscv_progbuf_exec_result_t;
+
```

Comment:
Thank you for unifying the names (debug_buffer -> progbuf) in many places in the code.

One last nitpick - `RISCV_DBGBUF_*` items in this enum can be renamed to `RISCV_PROGBUF_*`, too.

```suggestion
typedef enum {
	RISCV_PROGBUF_EXEC_RESULT_NOT_EXECUTED,
	RISCV_PROGBUF_EXEC_RESULT_UNKNOWN,
	RISCV_PROGBUF_EXEC_RESULT_EXCEPTION,
	RISCV_PROGBUF_EXEC_RESULT_UNKNOWN_ERROR,
	RISCV_PROGBUF_EXEC_RESULT_SUCCESS
} riscv_progbuf_exec_result_t;

```

Reply:
strange, I though I did that ...

Reply:
Random comment: It is much easier to review changes if you make a separate PR for renaming a bunch of stuff without changing any functionality.

Reply:
> Random comment: It is much easier to review changes if you make a separate PR for renaming a bunch of stuff without changing any functionality.

I moved these renames to a separate change set specifically to simplify the review. If this is still incontinent, I can factor-out this in a separate MR.

Reply:
@timsifive I've splitted the whole change set into 2 distinct commits: one with functional changes the other is with the rename of `dbgbuf->progbuf`. I've tested them separately.  If necessary, I can move the second changeset to a separate MR.

@JanMatCodasip : addressed.

Reply:
Thanks.